Makefile

```diff
@@ -192,7 +192,7 @@ tests/testreport/testreport: tests/testreport/testreport.go
 .PHONY: test-unit
 test-unit: tests/testreport/testreport
 	$(GO_TEST) -v -tags "$(STORAGETAGS) $(SECURITYTAGS)" -cover $(RACEFLAGS) $(shell $(GO) list ./... | grep -v vendor | grep -v tests | grep -v cmd | grep -v chroot | grep -v copier) -timeout 45m
-	$(GO_TEST) -v -tags "$(STORAGETAGS) $(SECURITYTAGS)"        $(RACEFLAGS) ./chroot ./copier -timeout 45m
+	$(GO_TEST) -v -tags "$(STORAGETAGS) $(SECURITYTAGS)"        $(RACEFLAGS) ./chroot ./copier -timeout 60m
```

Comment:
Note this unrelated change. The tests keep timing out, so I just bumped the maximum.

One recent success took 47m.

It would probably be worthwhile to spend some time speeding up the tests, but that’s not this PR.